custom_components/kia_uvo/KiaUvoApiEU.py

```diff
@@ -28,8 +28,9 @@ def __init__(
         region: int,
         brand: int,
         use_email_with_geocode_api: bool = False,
+        pin: int,
     ):
-        super().__init__(username, password, region, brand, use_email_with_geocode_api)
+        super().__init__(username, password, region, brand, use_email_with_geocode_api, pin)
```

Comment:
keep the correct constructor signature, pin should come after `password` as it is in CA implementation

Reply:
Done, I know you are making changes anyway but good practice for me.